pkg/virt-api/rest/expand.go

```diff
@@ -49,6 +49,7 @@ func (app *SubresourceAPIApp) ExpandSpecRequestHandler(request *restful.Request,
 		return
 	}
 
+	vm.Namespace = request.PathParameter("namespace")
```

Comment:
I wonder if we should reject the request if this isn't already the case in the provided VM?

Reply:
yes, let's reject the request if somehow the `namespace` parameter doesn't exist, even if it is kind of a redundant check.

While I don't think it's technically possible for the `namespace` parameter not to exist at this point, I think this is another layer of projection for the future. Code changes in the future, which can alter assumptions we once had about what is possible or not at a certain point in the code.